test/webaudio/audioworklet.c

```diff
@@ -80,8 +80,7 @@ bool main_thread_tls_access(double time, void *userData) {
 // added to the Worklet global scope.
 void AudioWorkletProcessorCreated(EMSCRIPTEN_WEBAUDIO_T audioContext, bool success, void *userData) {
   if (!success) {
-    emscripten_out("Stopped in AudioWorkletProcessorCreated");
-    assert(0);
+    assert("Stopped in AudioWorkletProcessorCreated" && success);
     return;
   }
```

Comment:
I would hop you can just do `assert(success && "Stopped in AudioWorkletProcessorCreated");` without the extra if / return.

Reply:
In this case, yes. I wasn't sure about code which also serves as examples that would be built without assertions (and would simply fail). But I'll change this.